scripts/20210517_validation-object/script.js

```diff
@@ -0,0 +1,283 @@
+/* eslint-disable */
+// Scripts for #408
+
+/**
+ * Part 1: Verify that the database is internally consistent today
+ * for all short text, long text and number fields.
+ * Expect 0 records for all scripts
+ */
+
+// a. customVal === customMin when selectedValidation === 'Minimum'
+// Expect 0 records
```

Comment:
do we need a fallback plan in case any of these are not 0? just tryna plan ahead so we don't get stuck when running this db migration on a weekend night. maybe we should decide what the source of truth should be for each case (short text, long text, number) - if it's always `customVal` then actually we don't even need to verify consistency, but if it's `customMin` or `customMax` then maybe we need to plan for the case where they're not equal

Reply:
I don't think this will happen - it is only legacy forms that we potentially need to worry about. Anyway this is something which should be done before the deployment itself - i'll test on prod secondary and update the status above